frontend_build/src/read_bundle_plugins.js

```diff
@@ -108,6 +108,21 @@ var readBundlePlugin = function(base_dir) {
     }
   });
 
+  // Create name to path mapping to allow translated json files to be copied into
+  // the correct static directory.
+
+  var namePathMapping = {};
+
+  bundles.forEach(function (bundle) {
+    namePathMapping[bundle.name] = path.resolve(path.dirname(bundle.output.path));
+  });
+
+  var locale_dir = path.join(base_dir, 'kolibri', 'locale')
+
+  mkdirp.sync(locale_dir);
+
+  fs.writeFileSync(path.join(locale_dir, 'pathMapping.json'), JSON.stringify(namePathMapping));
```

Comment:
It's specific to the build machine, so it maps from the plugin name, e.g. `learn_module` to the absolute path of the static dir, e.g. `/home/richard/github/kolibri/kolibri/plugins/learn/static`.